tasks/sep/027-holocene-system-config-upgrade-and-init-multi-chain/VALIDATION.md

```diff
@@ -0,0 +1,89 @@
+* **Before**: `0x0000000000000000000000000000000000000000000000000000000001c9c380`
+* **After**: `0x0000000000000000000000000000000000000000000a6fe00000000001c9c380`
+* **Meaning**: Updates the `basefeeScalar` and `blobbasefeeScalar` storage variables to `684000` (`cast td 0x000a6fe0`) and `0` respectively. These share a slot with the `gasLimit` which remains at `30000000` (`cast td 0x0000000001c9c380`). See the storage layout snapshot [here.](https://github.com/ethereum-optimism/optimism/blob/2073f4059bd806af3e8b76b820aa3fa0b42016d0/packages/contracts-bedrock/snapshots/storageLayout/SystemConfig.json#L58-L78)
+
+and 
+
+* **Key**: `0x0000000000000000000000000000000000000000000000000000000000000066`
+* **Before**: `0x00000000000000000000000000000000000000000000000000000000000a6fe0`
+* **After**: `0x01000000000000000000000000000000000000000000000000000000000a6fe0`
+* **Meaning**: Updates the `scalar` storage variable to reflect a scalar version of `1`. See the storage layout snapshot [here](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.8.0/packages/contracts-bedrock/snapshots/storageLayout/SystemConfig.json#L44-L50).
+
+### `0x15cd4f6e0CE3B4832B33cB9c6f6Fe6fc246754c2` (`SystemConfigProxy` for mode-sepolia)
+
+* **Key**: `0x0000000000000000000000000000000000000000000000000000000000000068`
+* **Before**: `0x0000000000000000000000000000000000000000000000000000000001c9c380`
+* **After**: `0x000000000000000000000000000000000008ee87000003d10000000001c9c380`
+* **Meaning**: Updates the `basefeeScalar` and `blobbasefeeScalar` storage variables to `977` (`cast td 0x000003d1`) and `585351` (` cast td 0x0008ee87`) respectively. These share a slot with the `gasLimit` which remains at `30000000` (`cast td 0x0000000001c9c380`). See the storage layout snapshot [here.](https://github.com/ethereum-optimism/optimism/blob/2073f4059bd806af3e8b76b820aa3fa0b42016d0/packages/contracts-bedrock/snapshots/storageLayout/SystemConfig.json#L58-L78)
+
+### `0xB54c7BFC223058773CF9b739cC5bd4095184Fb08` (`SystemConfigProxy` for zora-sepolia)
+
+* **Key**: `0x0000000000000000000000000000000000000000000000000000000000000068`
+* **Before**: `0x0000000000000000000000000000000000000000000000000000000001c9c380`
+* **After**: `0x00000000000000000000000000000000000941ad000003f40000000001c9c380`
+* **Meaning**: Updates the `basefeeScalar` and `blobbasefeeScalar` storage variables to `1012` (`cast td 0x000003f4`) and `606637` (cast td `0x000941ad`) respectively. These share a slot with the `gasLimit` which remains at `30000000` (`cast td 0x0000000001c9c380`).
+
+
+### `0x1Eb2fFc903729a0F03966B917003800b145F56E2` (`ProxyAdminOwner` for all chains in this task)
+
+* **Key**: `0x0000000000000000000000000000000000000000000000000000000000000005`
+* **Before**: `0x000000000000000000000000000000000000000000000000000000000000000f`
+* **After**: `0x0000000000000000000000000000000000000000000000000000000000000010`
```

Comment:
Invalid state validation here has been incremented. 
Need to recheck why and update the correct nonce expected.

Reply:
[85bca50](https://github.com/ethereum-optimism/superchain-ops/pull/418/commits/85bca50362715920d34ac96536de114b590f5fa9)